R/model.R

```diff
@@ -7,8 +7,9 @@
 #' @param pop population. See [squire::get_population]
 #' @param parameters parameters list.
 #'   See [squire::parameters_explicit_SEEIR]
+#' @param processes processes
 #' @export
-run_simulation <- function(pop, parameters) {
+run_simulation <- function(pop, parameters, processes) {
```

Comment:
Can we create the processes instead of the user? We don't expect users to change them.